tooling/Microsoft.VisualStudio.SecretManager/Microsoft.VisualStudio.SecretManager.csproj

```diff
@@ -37,7 +37,7 @@
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>
```

Comment:
@NTaylorMullen I think this is set to true in razor. Should be set to false IIUC